varcode/vcf.py

```diff
@@ -12,19 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# required so that 'import vcf' gets the global PyVCF package,
-# rather than our local vcf module
-from __future__ import absolute_import
```

Comment:
Since every module now has `absolute_import` so it's weird to explain why we're doing it in this case.